src/aiida/cmdline/commands/cmd_presto.py

```diff
@@ -0,0 +1,133 @@
+###########################################################################
+# Copyright (c), The AiiDA team. All rights reserved.                     #
+# This file is part of the AiiDA code.                                    #
+#                                                                         #
+# The code is hosted on GitHub at https://github.com/aiidateam/aiida-core #
+# For further information on the license, see the LICENSE.txt file        #
+# For further information please visit http://www.aiida.net               #
+###########################################################################
+"""``verdi presto`` command."""
+
+from __future__ import annotations
+
+import pathlib
+import re
+import typing as t
+
+import click
+
+from aiida.cmdline.commands.cmd_verdi import verdi
+from aiida.cmdline.utils import echo
+from aiida.manage.configuration import get_config_option
+
+DEFAULT_PROFILE_NAME_PREFIX: str = 'presto'
+
+
+def get_default_presto_profile_name():
+    from aiida.manage import get_config
+
+    profile_names = get_config().profile_names
+    indices = []
+
+    for profile_name in profile_names:
+        if match := re.search(r'presto[-]?(\d+)?', profile_name):
+            indices.append(match.group(1) or '0')
+
+    if not indices:
+        return DEFAULT_PROFILE_NAME_PREFIX
+
+    last_index = int(sorted(indices)[-1])
+
+    return f'{DEFAULT_PROFILE_NAME_PREFIX}-{last_index + 1}'
+
+
+@verdi.command('presto')
+@click.option(
+    '--profile-name',
+    default=lambda: get_default_presto_profile_name(),
+    show_default=True,
+    help=f'Name of the profile. By default, a unique name starting with `{DEFAULT_PROFILE_NAME_PREFIX}` is '
+    'automatically generated.',
+)
+@click.option(
+    '--email',
+    default=get_config_option('autofill.user.email') or 'aiida@localhost',
+    show_default=True,
+    help='Email of the default user.',
+)
+@click.pass_context
+def verdi_presto(ctx, profile_name, email):
+    """Set up a new profile in a jiffy.
+
+    This command aims to make setting up a new profile as easy as possible. It intentionally provides only a limited
+    amount of options to customize the profile and by default does not require any options to be specified at all. For
+    full control, please use `verdi profile setup`.
+
+    After running `verdi presto` you can immediately start using AiiDA without additional setup. The created profile
+    uses the `core.sqlite_dos` storage plugin which does not require any services, such as PostgreSQL. The broker
+    service RabbitMQ is also optional. The command tries to connect to it using default settings and configures it for
+    the profile if found. Otherwise, the profile is created without a broker configured. This _does_ mean, however, that
+    not all functionality of AiiDA is available most notably running the daemon and submitting processes to said daemon.
```

Comment:
```suggestion
    the profile if found. Otherwise, the profile is created without a broker, in which case some of the AiiDA functionality will be unavailable, 
    most notably running the daemon and submitting processes to said daemon.
```